tests/slsa_analyzer/checks/test_build_service_check.py

```diff
@@ -147,3 +205,5 @@ def test_build_service_check(self) -> None:
         ci_info["service"] = gitlab_ci
         maven_build_ci.dynamic_data["ci_services"] = [ci_info]
         assert check.run_check(maven_build_ci, check_result) == CheckResultType.FAILED
+
+        # TODO: Python module - maybe not for this context, just build
```

Comment:
Can you please elaborate?

Reply:
This was meant to be removed, done in [9c167f2](https://github.com/oracle-samples/macaron/pull/67/commits/9c167f271c3b880c3db95d6de5fc7bb51bdfcf3d).